src/scripts/backupScript.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# 
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+# Input the pathname for the desired backup directory
+# ie PATH="/home/<your_username>/path_to/database_dumps/dump.sql"
+# This path MUST exist
+
+# This could probably be programmatically populated. Currently needs to be set manually
+db_dump_path="/home/<your_username>/database_dumps" #INPUT REQUIRED
+
+# Generate a timestamp to append to the dump file. This line has errors at the moment.
```

Comment:
This is working fine for me. Is there still an issue you see?

Reply:
Apologies, this was a comment that we forgot to remove originally, there aren't any issues with this. Thank you for noticing that.